test/pkcs11_hsm_test/libp11_readhsm.c

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright © 2020, Andreas Jellinghaus <andreas@ionisiert.de>
+ * All rights reserved.
```

Comment:
This is a file which isn't written by you and under a different license?  If so, it will have to be declared separately or removed.

Reply:
Fixed!